database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -30,6 +30,13 @@ public function up()
      */
     public function down()
     {
-        Schema::dropIfExists('users');
+
+DB::statement('SET FOREIGN_KEY_CHECKS = 0');
+    Schema::dropIfExists('users');
+    DB::statement('SET FOREIGN_KEY_CHECKS = 1');
```

Comment:
This will break everything when we'll have at least 1 FK pointing to users. 

  What's the point of this at all?

Reply:
Forgot to remove that, from debug phase